rest-api-spec/api/termvector.json

```diff
@@ -18,8 +18,7 @@
         },
         "id" : {
            "type" : "string",
-           "description" : "The id of the document.",
-           "required" : true
```

Comment:
https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/api/bulk.json others just leave required out for parts where its `false` too.